.gitignore

```
@@ -138,4 +138,7 @@ lightning_logs/
 wandb/
 **.png
 **.pth
-**.ipynb
\ No newline at end of file
+**.ipynb
+datasets/dev_lvl7_robustness_muon_neutrino_0000.db
+examples/train_model.py
+examples/train_model.py
```

I would not add `examples/train_model.py` to .gitignore, as we might reasonably want to update it. I think it would be better to just not add local changes to this file to unrelated commits. Wrt. the other file, I think it it too specific to add to the .gitignore. I would suggest `**.db` or similar, if we don't ever want to add SQLite databases to the repo (which is probably pretty reasonable).
```suggestion
**.ipynb
**.db

```

---

This one is wholly related to my git inexperience, thinking it was just what was ignored locally on my end and what you write makes great sense.